src/routing/pages/probe/GleanDetails.svelte

```diff
@@ -285,7 +284,6 @@
   </div>
   <div class="probe-details-download">
     <div class="drawer-section drawer-section--end">
-      <SqlModal />
```

Comment:
Remove "View SQL query" feature for Glean probes for now, as without the Glam SQL there's nothing to show here. Once the SQL queries are done we'll add it back up.